src/components/map/RealtimeLayersService.js

```diff
@@ -90,11 +91,14 @@ goog.require('ga_map_service');
               // Layer Added
               newLayers.forEach(function(newLayer) {
                 var realTimeId = newLayer.bodId || newLayer.id;
-                if (realTimeLayersId.indexOf(realTimeId) === -1) {
-                  if (!newLayer.preview) {
-                    realTimeLayersId.push(realTimeId);
+                if (realTimeLayersId.indexOf(realTimeId) === -1 &&
+                    !newLayer.preview) {
+                  realTimeLayersId.push(realTimeId);
+                  if (newLayer.bodId) {
```

Comment:
It exists an `updateDelay` parameter which works with external KML layer. These external KML layer haven't `bodId` property.
Try to fix this. [hereis a test link](https://mf-geoadmin3.dev.bgdi.ch/ltpin/src/?lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-grau&X=196500.00&Y=624125.00&zoom=1&layers_params=updateDelay%3D3&lang=fr&layers=KML%7C%7Chttps:%2F%2Fpublic.int.bgdi.ch%2Fu1E0P4d_Tom7s7w8Ez4Kng)